akka-docs/src/main/paradox/stream/operators/Sink/collect.md

```diff
@@ -0,0 +1,27 @@
+# Sink.collect
+
+Collect all input elements using a Java 8 @javadoc[Collector](java.util.stream.Collector).
+
+@ref[Sink operators](../index.md#sink-operators)
+
+## Signature
+
+@apidoc[Sink.collect](Sink$) { java="#collect(akka.japi.function.Creator)" }
+
+
+## Description
+
+A @javadoc[Sink](akka.stream.javadsl.Sink) which materializes into a @javadoc[CompletionStage](java.util.concurrent.CompletionStage) 
+which will be completed with a result of the Java 8 @javadoc[Collector](java.util.stream.Collector) transformation and reduction operations.
+
+## Reactive Streams semantics
+
+@@@div { .callout }
+
+**cancels** never
```

Comment:
I think it will cancel if the collector apply throws, but is that not expected so not worth mentioning?

Reply:
I copy it from the `fold` one, and yes, I think should mention the exact behavior.